### PR TITLE
Item hierarchy spec

### PR DESCRIPTION
**Pull request by kathtran**

### CODE REVIEW

File: test/lib/schema/itemHierarchy.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+/* eslint-env mocha */
+let chai = require('chai');
```
Comment (ibilal) [inline test/lib/schema/itemHierarchy.js]: space after line 1

File: test/lib/schema/itemHierarchy.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+let expect = chai.expect;
+let dirtyChai = require('dirty-chai');
+let chaiAsPromised = require('chai-as-promised');
+let Ajv = require('ajv');
+
+let schema = require('../../../lib/schema/itemHierarchy');
```
Comment (ibilal) [inline test/lib/schema/itemHierarchy.js]: remove space line 7

**Comment by ibilal** [comment]
Besides spacing, LGTM

File: test/lib/schema/itemHierarchy.js (hunk 2 above)
Comment (RickyV33) [inline test/lib/schema/itemHierarchy.js]: We want to separate out or dev dependencies and our local code requires though

**Pull request merged**